osx/install_swift_binaries.sh

```diff
@@ -31,9 +31,24 @@ brew install wget > /dev/null || brew outdated wget > /dev/null || brew upgrade
 
 # Install Swift binaries
 # See http://apple.stackexchange.com/questions/72226/installing-pkg-with-terminal
-# TODO: Since Xcode now includes the Swift compiler and Xcode is included in the macOS image provided
-# by Travis CI, we could add logic here that checks whether the Swift binaries are already available
-wget https://swift.org/builds/$SNAPSHOT_TYPE/xcode/$SWIFT_SNAPSHOT/$SWIFT_SNAPSHOT-osx.pkg
-sudo installer -pkg $SWIFT_SNAPSHOT-osx.pkg -target /
-export PATH=/Library/Developer/Toolchains/swift-latest.xctoolchain/usr/bin:"${PATH}"
-rm $SWIFT_SNAPSHOT-osx.pkg
+
+# Set the var to be the version of swift that is intalled.
+SWIFT_PREINSTALL="swift-$(swift --version | awk '{print $4}')"
+echo "Preinstalled: $SWIFT_PREINSTALL"
+echo "Snapshot type: $SNAPSHOT_TYPE"
+echo "Swift version: $SWIFT_VERSION"
+echo "Default Swift: $DEFAULT_SWIFT"
+
+extra="-RELEASE"
+
+if [[ ${SWIFT_SNAPSHOT} == ${SWIFT_PREINSTALL} ]]; then
+  echo "Required Swift version is already installed, skipping download... A"
```

Comment:
Could you clean up these echo statements? There's some debugging logic  ('A', 'B') that aren't necessary.